dspace-api/src/main/resources/org/dspace/storage/rdbms/sqlmigration/postgres/V7.6_2023.03.17__Remove_unused_sequence.sql

```diff
@@ -0,0 +1,13 @@
+--
+-- The contents of this file are subject to the license and copyright
+-- detailed in the LICENSE and NOTICE files at the root of the source
+-- tree and available online at
+--
+-- http://www.dspace.org/license/
+--
+
+-----------------------------------------------------------------------------------
+-- Drop the 'history_seq' sequence (related table deleted at Dspace-1.5)
+-----------------------------------------------------------------------------------
+
+DROP SEQUENCE history_seq;
```

Comment:
@aroman-arvo : Since this is such an old sequence, it's *possible* others have deleted this manually. So, I'd highly recommend we add `IF EXISTS` to this statement.  Otherwise this will fail if a site already deleted this sequence.

Reply:
IF EXIST added...